recipes/deprecation-factory/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and paster from another recipe, but not really needed.
+{% set name = "deprecation-factory" %}
+{% set version = "0.1.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deprecation_factory-{{ version }}.tar.gz
+  # Get from pypi next to download files
+  sha256: 8bfa75fc5e9d914ca5346c0d3027997fdf89aaaec1509dc75877c834f738e39f
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
```

Comment:
This package looks like it could probably be `noarch: python`:

https://conda-forge.org/docs/meta.html#building-noarch-packages